interface/patient_file/summary/add_edit_issue.php

```diff
@@ -552,7 +552,6 @@ function outcomeClicked(cb) {
    var today = new Date();
    f.form_end.value = '' + (today.getYear() + 1900) + '-' +
     ("0" + (today.getMonth() + 1)).slice(-2) + '-' + ("0" + today.getDate()).slice(-2);
-   f.form_end.focus();
```

Comment:
👍 this does the trick. Please always describe your PR what it does a suitable name would be **Fix: issue title** no need placing the issue number cause github references issues when they are related or mentioned in PRs.
Approving this

Reply:
Okay, will keep that in mind

Reply:
Nice description like "Changed focus thingie on my first step to becoming a major daily contributor to LibreHealthEHR" or something like that is usually good to have.

Reply:
I have added a description to my pull request. Will remember to always do so.